pkg/golang/sum/parse_test.go

```diff
@@ -39,10 +39,15 @@ func TestParse(t *testing.T) {
 		t.Run(path.Base(v.file), func(t *testing.T) {
 			f, err := os.Open(v.file)
 			require.NoError(t, err)
+			defer f.Close()
 
 			got, _, err := NewParser().Parse(f)
 			require.NoError(t, err)
 
+			for i := range got {
+				got[i].ID = "" // Not compare IDs
+			}
```

Comment:
we can use `go list -m all | awk 'NR>1 {sub(/^v/, "", $2); printf("{ID: \""$1"@v"$2"\", Name: \""$1"\", Version: \""$2"\"},\n")}'
` to create ID.
Or is there another reason to skip  `IDs` check?

Reply:
ID is like a pseudo field generated by us. In this test, I just want to compare the fields shown by the Go official tool like `go list`. I've added another test.
https://github.com/aquasecurity/go-dep-parser/pull/176/commits/132ab47235b202895d1e3e220032ac62e0c4c8c5